server/model/pipeline.go

```diff
@@ -17,37 +17,37 @@ package model
-	Title               string            `json:"title"                   xorm:"build_title"`
-	Message             string            `json:"message"                 xorm:"TEXT 'build_message'"`
-	Timestamp           int64             `json:"timestamp"               xorm:"build_timestamp"`
-	Sender              string            `json:"sender"                  xorm:"build_sender"` // uses reported user for webhooks and name of cron for cron pipelines
-	Avatar              string            `json:"author_avatar"           xorm:"build_avatar"`
-	Email               string            `json:"author_email"            xorm:"build_email"`
-	Link                string            `json:"link_url"                xorm:"build_link"`
-	Signed              bool              `json:"signed"                  xorm:"build_signed"`   // deprecate
-	Verified            bool              `json:"verified"                xorm:"build_verified"` // deprecate
-	Reviewer            string            `json:"reviewed_by"             xorm:"build_reviewer"`
-	Reviewed            int64             `json:"reviewed_at"             xorm:"build_reviewed"`
+	Started             int64             `json:"started_at"              xorm:"pipeline_started"`
+	Finished            int64             `json:"finished_at"             xorm:"pipeline_finished"`
+	Deploy              string            `json:"deploy_to"               xorm:"pipeline_deploy"`
+	Commit              string            `json:"commit"                  xorm:"pipeline_commit"`
+	Branch              string            `json:"branch"                  xorm:"pipeline_branch"`
+	Ref                 string            `json:"ref"                     xorm:"pipeline_ref"`
+	Refspec             string            `json:"refspec"                 xorm:"pipeline_refspec"`
+	Remote              string            `json:"remote"                  xorm:"pipeline_remote"`
+	Title               string            `json:"title"                   xorm:"pipeline_title"`
+	Message             string            `json:"message"                 xorm:"TEXT 'pipeline_message'"`
+	Timestamp           int64             `json:"timestamp"               xorm:"pipeline_timestamp"`
+	Sender              string            `json:"sender"                  xorm:"pipeline_sender"` // uses reported user for webhooks and name of cron for cron pipelines
+	Avatar              string            `json:"author_avatar"           xorm:"pipeline_avatar"`
+	Email               string            `json:"author_email"            xorm:"pipeline_email"`
+	Link                string            `json:"link_url"                xorm:"pipeline_link"`
+	Signed              bool              `json:"signed"                  xorm:"pipeline_signed"`   // deprecate
+	Verified            bool              `json:"verified"                xorm:"pipeline_verified"` // deprecate
+	Reviewer            string            `json:"reviewed_by"             xorm:"pipeline_reviewer"`
```

Comment:
as we refactor that anyway ... do we wanta stick to one nameing schema?

I would drop the table prefix as you can `select table.field` ...